### PR TITLE
fix pkgconfig jsoncpp

### PR DESCRIPTION
**Pull request by diamant3**
They have the same value of `prefix` and `exec_prefix`

jsoncpp-pkgconfig

### CODE REVIEW

File: jsoncpp/PSPBUILD [hunk 1]
```diff
@@ -9,6 +9,14 @@ makedepends=()
 source=("git+https://github.com/open-source-parsers/jsoncpp#commit=5defb4ed1a4293b8e2bf641e16b156fb9de498cc")
 sha256sums=('SKIP')
 
+prepare() {
+    cd "$pkgname/pkg-config"
+    sed -i 's#@CMAKE_INSTALL_PREFIX@#${PSPDEV}/psp#' jsoncpp.pc.in
+    sed -i 's#@CMAKE_INSTALL_PREFIX@#${prefix}#' jsoncpp.pc.in
```
Comment (sharkwouter) [inline jsoncpp/PSPBUILD:15]: This line can be removed, it's not used.

File: jsoncpp/PSPBUILD [hunk 2]
```diff
@@ -9,6 +9,14 @@ makedepends=()
 source=("git+https://github.com/open-source-parsers/jsoncpp#commit=5defb4ed1a4293b8e2bf641e16b156fb9de498cc")
 sha256sums=('SKIP')
 
+prepare() {
+    cd "$pkgname/pkg-config"
+    sed -i 's#@CMAKE_INSTALL_PREFIX@#${PSPDEV}/psp#' jsoncpp.pc.in
```
Comment (sharkwouter) [inline jsoncpp/PSPBUILD:14]:
```suggestion
    sed -i 's#@CMAKE_INSTALL_PREFIX@#${PSPDEV}/psp#g' jsoncpp.pc.in
```
I'd add this just to make sure it applies to both instances, even though it already seems to.

**Review by sharkwouter — COMMENTED** [review]
Looks good, but I did notice that one of the statements in the prepare function does not apply.

File: jsoncpp/PSPBUILD (hunk 1 above)
Comment (diamant3) [reply jsoncpp/PSPBUILD:15]: Thanks, got it

**Comment by diamant3** [comment]
Done. Can I request a review again? thanks

**Pull request merged**